discovery-provider/Dockerfile

```diff
@@ -107,7 +107,7 @@ ENV audius_loggly_tags=$audius_loggly_tags
 ENV PROMETHEUS_MULTIPROC_DIR /prometheus_data
 RUN mkdir -p ${PROMETHEUS_MULTIPROC_DIR}
 
-HEALTHCHECK --interval=5s --timeout=5s \
-    CMD curl -f http://localhost:5000/health_check || exit 1
+HEALTHCHECK --interval=5s --timeout=5s --retries=12 \
```

Comment:
I added the retry endpoint, would that be useful here? https://github.com/AudiusProject/audius-protocol/pull/3031